test/e2e/common_test.go

```diff
@@ -102,6 +102,25 @@ func TestLibpod(t *testing.T) {
 	RunSpecs(t, "Libpod Suite")
 }
 
+var (
+	tempdir    string
+	err        error
+	podmanTest *PodmanTestIntegration
+
+	_ = BeforeEach(func() {
+		tempdir, err = CreateTempDirInTempDir()
+		Expect(err).ToNot(HaveOccurred())
+		podmanTest = PodmanTestCreate(tempdir)
+		podmanTest.Setup()
+	})
+
+	_ = AfterEach(func() {
+		podmanTest.Cleanup()
```

Comment:
In the `volume_*` tests (far below at bottom), the cleanup function is `CleanupVolume()`. Your PR loses that. I'm not sure how best to address it, it depends on the ordering of `AfterEach()`.

Reply:
very good eyes, I will fix this.